### DISCUSSION TITLE
Welcome to Recommended-Bets-By-Email-MLB Discussions!

### DISCUSSION THREAD

**Discussion by callmevojtko** [callmevojtko/Recommended-Bets-By-Email-MLB#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by paulshryack** [comment]
how do i run your bot please. is it still working

**Comment by gomezj161** [comment]
Can you offer a step by step video to run the program. Im having error messages from importing the modules
